worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -282,7 +292,7 @@ private MemoryManager(CelebornConf conf, StorageManager storageManager, Abstract
         Utils.bytesToString(readBufferThreshold),
         Utils.bytesToString(readBufferTarget),
         Utils.bytesToString(memoryFileStorageThreshold),
-        resumeRatio);
+        directMemoryResumeRatio);
```

Comment:
You can add pinned memory resume ratio here. It is an important parameter for memory manager.